### PR TITLE
Updating for CAS changes and new select a driver section

### PR DESCRIPTION
**Pull request by moxiegirl**
Adding changes requested by @jamtur01 and wrapping to 80 chars
Adding typo fixes and replacing tabs with 4xspaces
Closes and fixes #19240
Updating with content addressable storage model changes

Signed-off-by: Nigel nigelpoulton@hotmail.com

### CODE REVIEW

**Comment by moxiegirl** [comment]
@thaJeztah LGTM from me.  I just carried this.  No changes.   Already been reviewed. Can you slack me if you need something else on this.  We have to get it in.

File: docs/userguide/storagedriver/aufs-driver.md [hunk 1]
```diff
@@ -10,184 +10,203 @@ parent = "engine_driver"
...
 
 Alternatively, you can edit the Docker config file and add the
 `--storage-driver=aufs` option to the `DOCKER_OPTS` line.
 
-```bash
-# Use DOCKER_OPTS to modify the daemon startup options.
-DOCKER_OPTS="--storage-driver=aufs"
-```
+    # Use DOCKER_OPTS to modify the daemon startup options.
+    DOCKER_OPTS="--storage-driver=aufs"
```
Comment (thaJeztah) [inline docs/userguide/storagedriver/aufs-driver.md]: We should have a follow up PR to be more generic, and not use `DOCKER_OPTS` here; most systems now use systemd, and we don't use `DOCKER_OPTS` on systemd

**Comment by thaJeztah** [comment]
LGTM, thanks @nigelpoulton!

**Comment by moxiegirl** [comment]
Thanks @thaJeztah for the assist.

**Pull request merged**